CHANGELOG.asciidoc

```diff
@@ -35,14 +35,21 @@ endif::[]
 [[release-notes-4.x]]
 === Ruby Agent version 4.x
 
+[[release-notes-4.6.0]]
+==== 4.6.0
+
+[float]
+===== Changed
+
+- When the configuration value for `log_path` is set, override the `logger` to point to that path instead of using e.g. Rails logger {pull}1247[#1247]
+
 [[release-notes-4.5.1]]
 ==== 4.5.1
 
 [float]
 ===== Changed
 
 - Update elasticsearch spy to use new transport gem name {pull}1257[#1257]
-- Standardize placeholder for phone numbers as [PHONENUMBER] per https://github.com/elastic/apm/blob/main/specs/agents/tracing-instrumentation-aws.md {pull}1246[#1246]
```

Comment:
Just make sure that we you rebase and merge, this changelog entry is not mistakenly removed.